config/webpack-config.js

```diff
@@ -22,7 +22,7 @@ const COMMON_MODULE_CONFIG = {
       options: {
         presets: [ [ 'env', {
           targets: {
-            browsers: environment.getSupportedBrowserList( 'js' )
+            browsers: browserList.last2IE9up
```

Comment:
We should have a standardized way of naming constants. Shouldn't this be `LAST_2_IE_9_UP`